owledge/db/seeds.rb

```diff
@@ -21,4 +27,34 @@
                   that following those directions will never form a closed loop.
                }.join(" "),
                parent_id: graph_theory.id
-             }])
\ No newline at end of file
+             }])
+
+             
+Note.create([{
```

Comment:
Good that it's in here, but it is still too verbose. Try:

```
Note.create([{
          title: 'Sorting Algorithms 3',
          description: %w{
            In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order.
          }.join(" "),
          parent_id: sorting.id

}, {
          title: 'Sorting Algorithms 4',
          description: %w{
            In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order.
          }.join(" "),
          parent_id: sorting.id

}])
```

Reply:
Didnt realize it was separate thought I had merged them, my bad! Made the change now